python/mxnet/test_utils.py

```diff
@@ -1407,12 +1410,15 @@ def download(url, fname=None, dirname=None, overwrite=False):
                 if exc.errno != errno.EEXIST:
                     raise OSError('failed to create ' + dirname)
 
-    if not overwrite and os.path.exists(fname):
+    if not overwrite and os.path.exists(fname) and not version_tag:
```

Comment:
how do you plan to handle if i give a different eTag

Reply:
Have handled it in line 1419-1420